packages/components/affix/src/affix.vue

```diff
@@ -131,6 +131,7 @@ export default defineComponent({
 
     useEventListener(scrollContainer, 'scroll', onScroll)
     useResizeObserver(root, () => update())
+    useResizeObserver(target, () => update())
```

Comment:
I'm just being curious. Why can't we just use `useResizeObserver(target, update)` here?

Reply:
To make it clear that this is a callback